app.js

```diff
@@ -12,6 +12,7 @@ const swaggerUI = require('swagger-ui-express')
 const swaggerOptions = require('~/swagger-settings')
 const example = require('~/routes/example')
 const auth = require('~/routes/auth')
+const { createError, handleError } = require('~/utils/errors')
 
```

Comment:
{
status: 400 404....,
message: 'User not found',
errorCode: 'USER_NOT_FOUND'
}

Reply:
done

Reply:
done